google/cloud/security/common/data_access/csv_writer.py

```diff
@@ -116,6 +116,20 @@
     'buckets'
 ]
 
+BUCKETS_ACL_FIELDNAMES = [
```

Comment:
It's fine to leave a TODO here for this. But at some point the entier group should circle back and create some pydoc in these definitions that maps our custom field naming to the field names in the [resource objects](https://cloud.google.com/storage/docs/json_api/v1/buckets#resource).